Description: Use /usr/bin/nodejs to supply Node.js
 Debian does not install Node.js to the same location as upstream
Forwarded: not-needed
Author: Geza Kovacs <gkovacs@mit.edu>
--- a/bin/cake
+++ b/bin/cake
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env nodejs
 
 var path = require('path');
 var fs = require('fs');
--- a/bin/coffee
+++ b/bin/coffee
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env nodejs
 
 var path = require('path');
 var fs = require('fs');
